docs/production/install.md

```diff
@@ -61,10 +61,24 @@ follows:
 
 ```bash
 sudo -s  # If not already root
-./zulip-server-*/scripts/setup/install --certbot \
+./zulip-server-*/scripts/setup/install --push-notifications --certbot \
     --email=YOUR_EMAIL --hostname=YOUR_HOSTNAME
 ```
 
+:::{note}
+This will register your server for push notifications, due to the
+`--push-notifications` flag. Upon running the command, you will immediately be
+prompted to agree to the Terms of Service.
+
+If you do not want to register for push notifications at this time, you can
+remove the `--push-notifications` flag from the command or pass
+`--no-push-notifications` explicitly.
```

Comment:
Similar to as noted below for the CLI output, I think we should link to https://zulip.readthedocs.io/en/latest/production/mobile-push-notifications.html#why-a-push-notification-service-is-necessary or similar somewhere in here, so folks can understand why a TOS is necessary for the push notifications.

Reply:
Added this to the installer output
```
    echo "For details on why a centralized push notification service is necessary, see:"
    echo "    <https://zulip.readthedocs.io/en/latest/production/mobile-push-notifications.html#why-a-push-notification-service-is-necessary>"
```

and this to the doc
```
  A centralized push notification service is needed for delivering notifications
  to mobile devices; see [why this is necessary](mobile-push-notifications.md#why-a-push-notification-service-is-necessary).
```
```